### PR TITLE
Heretics

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

Added [Heretics](https://tgstation13.org/wiki/Heretic)

draft PR indicates that i'm working on it
also used for fixing merge conflicts if any rise.

be patient :godo:

## Why / Balance

gaming

## Technical details

will spawn on par with thieves with a 20% chance each round
~6666 LINES!!! very important~ not anymore...

## Media
image

:cl:
- add: Forgotten, devoured, gutted. Humanity has forgotten the eldritch forces of decay, but the mansus veil has weakened. We will make them taste fear again.

### CODE REVIEW

**Comment by cheki-ss14** [comment]
So based. Incredibly based even. Hairline is returning.

**Comment by yglop** [comment]
> Shitcode, this is hardcoded, erm shitcode shitcode.

Cry about it.

**Comment by Verbalase** [comment]
> > Shitcode, this is hardcoded, erm shitcode shitcode.
> 
> Cry about it.

repo banned

**Comment by a-person5660** [comment]
based PR, good luck getting this anywhere near wizden without them putting you in PR hell for all the shitcode tho :trollface:

**Comment by whateverusername0** [comment]
if you take a look at it it's mostly in yaml :trollface: therefore the shitcode is nonexistent

**Comment by Verbalase** [comment]
> if you take a look at it it's mostly in yaml :trollface: therefore the shitcode is nonexistent

oh boy let’s see what every maintainer to exist says

**Comment by whateverusername0** [comment]
> oh boy let’s see what every maintainer to exist says

i say it's good :trollface:

**Comment by whateverusername0** [comment]
image
nerd ass check

**Comment by whateverusername0** [comment]
damn. i guess making ritual behaviors server only fucked up yaml linting

**Comment by whateverusername0** [comment]
everything is working in game, it's just that yaml shitter is complaining
finna roll some fixes in the next couple hours

**Comment by whateverusername0** [comment]
yeah :trollface:

**Comment by whateverusername0** [comment]
aw fuck

**Comment by whateverusername0** [comment]
image
these checks are just bullshit. everything works in game like a charm, yet this mf comes in to ruin the checks

**Comment by whateverusername0** [comment]
these checks are killing my sanity

**Comment by lanse12** [comment]
blade path when

**Comment by whateverusername0** [comment]
image
great. how the checks say nothing. fantastic

**Comment by whateverusername0** [comment]
what the fuck is this?
image

**Comment by whateverusername0** [comment]
@Piras314 
**6666 lines**
it's a sign :trollface:

**Comment by whateverusername0** [comment]
all checks massive W

**Comment by jorgun** [comment]
QA testing:
Ritual selection is a bit odd. i think id wish this to be a verb whit select ritual. its easy to missckick and change ritual
amber focus can be butchered for 2  cloth whitch is wired.
final flesh monster can eat walls, like a dragon. i get eating corpses and punching hard but eating metal?
ending the round: this might be becuse of "dev" but is there any automatic round ender.
i keep forgeting what the rituals does, if there be a way to add a new tab in the spellbook whit aquired spells/skills.
no way of removing a ritual rune . somone can spam thes across the station whit no way of removal/cleaning it up.

**Comment by lanse12** [comment]
> QA testing: Ritual selection is a bit odd. i think id wish this to be a verb whit select ritual. its easy to missckick and change ritual amber focus can be butchered for 2 cloth whitch is wired. final flesh monster can eat walls, like a dragon. i get eating corpses and punching hard but eating metal? ending the round: this might be becuse of "dev" but is there any automatic round ender. i keep forgeting what the rituals does, if there be a way to add a new tab in the spellbook whit aquired spells/skills. no way of removing a ritual rune . somone can spam thes across the station whit no way of removal/cleaning it up.

i am pretty sure that u can't remove them on tg (or you probably don't because there's no reason to remove them)

**Comment by whateverusername0** [comment]
@jorgun responding to my twitter allegations

> Ritual selection is a bit odd. i think id wish this to be a verb whit select ritual.

> its easy to missckick and change ritual

you can just always change it back, there's no cooldown on that

but i get what you're saying, rit selection needs a UI, but as of now i can't do it because i simply don't know how. if we ever get a guy who's somewhat decent in ss14 UI development sure why not, but as of now 🤷‍♂️ sorry man, gotta thug it out.

> amber focus can be butchered for 2 cloth whitch is wired.

shouldn't be a problem :trollface: 
how did you come up with butchering it even?

> final flesh monster can eat walls, like a dragon. i get eating corpses and punching hard but eating metal?

yeah. his punches deal a lot of structural, so why not consume walls too? he's just like the dragon but eculty.

> ending the round: this might be becuse of "dev" but is there any automatic round ender.

heretic is a side antagonist (like thief) which is never supposed to end the round automatically, and there's no special gamemode explicitly for him.

because heretics are not supposed to be in groups. that's why i made him spawn once per 20 people, it's so that these other 19 people could somewhat deal with one heretic if crew teams up.

also when a heretic ascends **everyone** gets a big ass announcement stating "HI CHAT, I'M SOSA, TRY AND KILL ME IF YOU CAN", so it becomes a giant sesh of validhunting.

in conclusion, not a problem.

> i keep forgeting what the rituals does, if there be a way to add a new tab in the spellbook whit aquired spells/skills. no way of removing a ritual rune .

yeah that might be a problem. perhaps we improve the goob wiki page? or you just have to pay more attention when playing heretic. either way you hafta thug it out for now until i find a better way

> somone can spam thes across the station whit no way of removal/cleaning it up.

there's no reason to remove it because it's heretics exclusive.
also it takes like 15 seconds to draw a rune. if someone gets caught drawing it they're automatically valid and instantly get a hypothetical killsign above their head. so that's not an issue.

**Comment by a-person5660** [comment]
> > i keep forgeting what the rituals does, if there be a way to add a new tab in the spellbook whit aquired spells/skills. no way of removing a ritual rune .
> 
> yeah that might be a problem. perhaps we improve the goob wiki page? or you just have to pay more attention when playing heretic. either way you hafta thug it out for now until i find a better way

the guidebook in question:

**Comment by a-person5660** [comment]
L

**Comment by whateverusername0** [comment]
WHO

**Review by Piras314 — APPROVED** [review]

**Comment by ElPidor** [comment]
Go my child, LIVE!!!
image
